pcsx2/GS/Renderers/HW/GSRendererHW.cpp

```diff
@@ -2252,7 +2311,7 @@ void GSRendererHW::Draw()
 			GL_INS("WARNING: Possible misdetection of effect, texture shuffle is %s", m_texture_shuffle ? "Enabled" : "Disabled");
 		}
 
-		if (m_texture_shuffle && IsSplitTextureShuffle())
+		if (m_texture_shuffle && IsSplitTextureShuffle(rt ? rt->m_TEX0.TBW : m_cached_ctx.FRAME.FBW))
```

Comment:
I don't think rt should ever be null if texture_shuffle is true?

Reply:
Probably not, it was more just to save my ass in case of it being null and crashing xD

I can remove the check